docs/config.md

```diff
@@ -51,6 +51,22 @@ Similar to the wandb config above, these keyword parameters are fed directly int
 * `logging_steps`: How often evaluation is run using the evaluation dataset.
 * `save_frequency`: The frequency at which checkpointing occurs. This must be between 0 and 1.
 
+
+### Sampling during Training
+
+To disable sampling during training, delete the entire "sampling" section.
```

Comment:
Are we "deleting" the section or just commenting out?

Reply:
"Comment out" might be sufficient, as it allows the user to easily re-enabled the sampling engine as needed.